cmd/crictl/image.go

```diff
@@ -164,6 +171,27 @@ var listImageCommand = &cli.Command{
 			}
 		}
 		for _, image := range r.Images {
+			// don't show intermediate images if showall is not set
+			if !showAll {
+				status, err := ImageStatus(imageClient, image.Id, true)
+				if err != nil {
+					return err
+				}
+
+				if status.GetInfo() != nil {
+					var info map[string]interface{}
+					err := json.Unmarshal([]byte(status.Info["info"]), &info)
+					if err != nil {
+						return err
+					}
+
+					//if showall is not set, do not show images where intermediate is set
+					if info["Intermediate"] == true {
```

Comment:
where did this value come from?

Reply:
this PR goes with another PR in cri-o, which adds an intermediate flag to the image's info 
https://github.com/cri-o/cri-o/pull/3892/files#diff-401dc45799b30a238ffacfa0faed4248R103